bsp4j/src/main/java/ch/epfl/scala/bsp4j/JavaExtension.xtend

```diff
@@ -0,0 +1,105 @@
+package ch.epfl.scala.bsp4j
+
+import java.util.List
+import com.google.gson.annotations.SerializedName
+import org.eclipse.lsp4j.jsonrpc.validation.NonNull
+import org.eclipse.lsp4j.generator.JsonRpcData
+
+@JsonRpcData
+class JavaBuildTarget {
+  @NonNull String javaOrganization
```

Comment:
does organization make sense in Java? In scala we have it to differentiate different compiler flavors

Reply:
Ok, I misunderstood it's purpose. We can remove it then.